exposee-app/models/wallet.js

```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "../database.js";
+
+export const Wallet = sequelize.define("Wallet", {
+    user_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    },
+    amount: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    },
+    deductions: {
```

Comment:
this table could just be the amount without deductions and addition, but I'm OK to leave them in and not use them.